app/renderer/components/main/loginRequired.js

```diff
@@ -133,26 +133,23 @@ class LoginRequired extends React.Component {
                   />
                 </div>
               </div>
-              : null
             }
           </div>
         </CommonFormSection>
         <CommonFormButtonWrapper>
           <Button l10nId='cancel' className='whiteButton' onClick={this.onClose} />
-          <Button l10nId='ok' className='primaryButton' onClick={this.onSave.bind(this)} />
+          <Button l10nId='ok' className='primaryButton' onClick={this.onSave} />
         </CommonFormButtonWrapper>
       </CommonForm>
     </Dialog>
   }
 }
 
-LoginRequired.propTypes = { frameProps: PropTypes.object }
+module.exports = LoginRequired
```

Comment:
In all components we have this like that, so I would leave it for now and we can move it all to the bottom in a different PR